Controller/Payment/Webhook.php

```diff
@@ -472,7 +475,7 @@ protected function orderPaid(array $post)
                                 $this->orderSender->send($order);
                                 $this->checkoutSession->unsRazorpayMailSentOnSuccess();
                             }
-                            catch (\Magento\Framework\Exception\MailException $exception)
+                            catch (\Magento\Framework\Exception\MailException $e)
                             {
                                 $this->logger->critical($e);
```

Comment:
@nikhilsharma1100 Please use `$e->getMessage()` to log exception message.

Reply:
okay